### PR TITLE
Sync groups between desc and javascript snippet

### PR DESCRIPTION
**Pull request by bartocc**
The 9 groups were not reflected in the javascript snippet

### CODE REVIEW

**Comment by michalsnik** [comment]
Hi @bartocc, thanks for pointing this.

One thing - controller's default properties are: https://github.com/bartocc/eslint-plugin-ember/blob/master/lib/utils/ember.js#L277
and custom is for example given `attitude`

**Comment by michalsnik** [comment]
Hey, can you please update this PR so I can merge it @bartocc ? :)

**Comment by bartocc** [comment]
Rebased @michalsnik

File: docs/rules/order-in-controllers.md [hunk 1]
```diff
@@ -53,33 +53,36 @@ export default Controller.extend({
...
-  // 2. Default route's properties
+  // 2. Query params
   queryParams: ['view'],
 
-  // 3. Custom properties
+  // 3. Default controller's properties
   attitude: 10,
+  
+  // 4. Custom properties
+  ???
```
Comment (michalsnik) [inline docs/rules/order-in-controllers.md]: Custom property should be:
```
attitude: 10,
```

File: docs/rules/order-in-controllers.md [hunk 2]
```diff
@@ -53,33 +53,36 @@ export default Controller.extend({
   // 1. Services
   currentUser: service(),
 
-  // 2. Default route's properties
+  // 2. Query params
   queryParams: ['view'],
 
-  // 3. Custom properties
+  // 3. Default controller's properties
   attitude: 10,
```
Comment (michalsnik) [inline docs/rules/order-in-controllers.md]: default controller's property should be for example:
```
concatenatedProperties: ['concatenatedProperty']
```

**Comment by michalsnik** [comment]
up @bartocc

**Comment by bartocc** [comment]
@michalsnik I've made the modifications and rebased again

**Pull request merged**